tests/util.ts

```diff
@@ -49,6 +50,11 @@ export function starknetCompile(
   );
 }
 
+export function cairoTest(cairoProjectPath: string): Promise<{ stdout: string; stderr: string }> {
+  const cairoTestBin = path.resolve(__dirname, '..', 'cairo1', getPlatform(), 'bin', 'cairo-test');
```

Comment:
Use `BASE_PATH`. Perhaps we should make a function within warp that would construct the path on top of `BASE_PATH`? Seems like a cleaner option than always doing `path.resolve(BASE_PATH, ...)`